docs/developer/payments/deprecated.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Deprecated (Legacy)
+sidebar_label: Deprecated
+---
+
+## Payment Plugin
+
+Using PaymentPlugin is explained in [Checkout](/developer/checkout/finalizing.mdx) section.
+
+### Actions Saleor 3.4
+
+If your app should also process payment request actions (like charge, refund, or CANCEL) triggered by staff users on the Saleor dashboard side or by the app,
+make sure that your app subscribes to the [`transaction-action-request`](developer/extending/webhooks/sample-webhook-payloads.mdx#transaction-action-request) webhook.
+
+The chart below shows a workflow for handling a `refund` request.
+
+```mermaid
+sequenceDiagram
+    Staff->>+Dashboard: Click refund button<br> for single transaction
+    Dashboard->>+Saleor: Call transactionActionRequest<br> with action - refund
+    Saleor -->>- Dashboard: OK
+    Dashboard -->>- Staff: Triggered
+    Saleor ->>+ App: Send webhook<br>transaction-action-request<br> with all details related<br> to action and transaction
+    App -->>- Saleor: OK
+    App ->>+ PaymentProvider: Execute refund <br>action
+    PaymentProvider -->>- App: Refunded
+    App ->>+ Saleor: Call transactionUpdate <br>to update transaction with<br> a result received after <br>requested action
+    Saleor -->>- App: Updated transaction
+```
+
+### Creating transactions in Saleor 3.4
```

Comment:
`transactionCreate` / `transactionUpdate` are mutations that you will use, to cover more advanced cases, where you can't use transactionInitialize. It is also used on dashboard to create "manual transaction"

Reply:
@korycins do we even need to document 3.4 differences? It is quite old and the full path is probably not documented anyway?

Reply:
This is the section that I moved to a separate document, to avoid cluttering the guide with quite old examples. https://docs.saleor.io/developer/payments#saleor-34-deprecated

I suggest to remove it completely

Reply:
@korycins I have removed deprecated section